test/unit/idex/identity-resolver-nocache.spec.js

```diff
@@ -85,6 +85,19 @@ describe('IdentityResolver without cache', () => {
     requestToComplete.respond(200, { 'Content-Type': 'application/json' }, JSON.stringify(response))
   })
 
+  it('should attach publisher id', function (done) {
+    const response = { id: 112233 }
+    const identityResolver = IdentityResolver({ peopleVerifiedId: '987', identityResolutionConfig: { publisherId: '123' } }, calls)
```

Comment:
am not sure if it is the important, but should the publisher Id be a number, should we check it with this format? or it doesn't matter?
```scala
    identityResolutionConfig: { publisherId: 123 } 
```

Reply:
It does not matter. It is appended to the query part of the URL as is without any assumptions about the type.

Reply:
Changed it since we generate the config with a number.